content/en/logs/guide/logs-rbac-permissions.md

```diff
@@ -17,9 +17,9 @@ Once you've created [RBAC roles for logs][1], assign or remove [permissions][2]
 {{< tabs >}}
 {{% tab "UI" %}}
 
-Assign or remove permission to a role directly by [updating the role in the Datadog application][1].
+Assign or remove permission to a role directly by [updating the role on the Datadog site][1].
 
-{{< img src="account_management/rbac/logs_permissions.png" alt="Logs Permissions"  style="width:75%;" >}}
+{{< img src="account_management/rbac/logs_permissions.png" alt="Logs Permissions" style="width:75%;" >}}
```

Comment:
This screenshot is (or will be) out of date. But we don't have a final screenshot available yet. Consider removing it?